tezos/messages/src/p2p/encoding/version.rs

```diff
@@ -54,3 +55,11 @@ impl PartialEq for NetworkVersion {
             && self.p2p_version == other.p2p_version
     }
 }
+
+impl Hash for NetworkVersion {
```

Comment:
@mambisi 
does cached needs this?

Reply:
The NetworkVersion is used as key for cache which requires keys to `impl` `Hash`

Reply:
aha, ok, I see `get_node_version`, well I guess, this does not need to be cached, but ok